### DISCUSSION TITLE
Error when trying to authentica pro

### DISCUSSION THREAD

**Discussion by woutvdpol** [Laravel-Backpack/community-forum#696] [Q&A (Help)] [answered]
Hi I am trying to get a colleague of mine to get Laravel Backpack to work on his laptop, I am using Mac and he uses windows.
When trying to authenticate the pro version using `php artisan backpack:require:pro` and then putting in our credentials from the site, he gets the following error: 'Permission denied. Could not authenticate the credentials.'
He also gets the same error when he uses my auth.json.

Do you guys have tips to fix it or a solution?
Thx

**Comment by jcastroa87** [comment]
Hello @woutvdpol could you please check your license is exactly for the version you are try to install, didnt copy any white space, etc.

Could you send us a screenshot if the problem persists.

Cheers.

**Reply by woutvdpol** [reply]
We have a license for: 'backpack/pro' and trying to authenticate with 'php artisan backpack:require:pro'.
This is the first time he is starting a laravel project, so I thought it was maybe that, but I dont think it is.
![afbeelding](https://github.com/Laravel-Backpack/community-forum/assets/25347211/af8637cf-80a6-41f8-97d1-6f638607c7f3)

**Reply by jcastroa87** [reply]
Did you own v6 licence? because by default will try to install latest version, so if dont, lock on your composer.json the version you own and try to install again.

Cheers.

**Reply by tabacitu** [reply, marked as answer]
Yup, I would think it's the above too. 90% of the time when people get this error, it's because they have access to everything until 2.0.8 (for example) but they're trying to install the latest (at this time 2.0.13). To see if that's the case for you, go to [your Tokens page](https://backpackforlaravel.com/user/tokens) and see if the token no longer has access to PRO.

If that's the case, you can:
- (a) buy more access (purchase again, it will add 12 more months of updates to the same token);
- (b) lock your `composer.json` so it downloads the last version you have access to (eg. `"backpack/pro":"2.0.8"`);

When you try to install a version you don't have access to, Backpack will send the token owner an email, telling them exactly what version they should lock to.

--

If the above isn't it, please send an email to hello@backpackforlaravel.com with your screenshot and token name, we might be able to see what has happened from our logs.

Cheers!

**Comment by jcastroa87** [comment]
Hello @woutvdpol you was it possible for you to move forward with this problem?

Cheers.

**Reply by woutvdpol** [reply]
Yeah we found hte problem after our emails, thanks!
